docs/alarms.md

```diff
@@ -0,0 +1,13 @@
+### MaxPendingTime
+
+#### What
+
+Time between tasks getting created and actually starting (i.e. staying in PENDING state) has gone above 120 seconds.
+
+#### Problem
+
+Long pending times could contribute to backup of minutes to hours. When the cluster is under heavy load and there is throttling occuring between the ecs-agent and the Agent Control Service (ACS), there is often be a buildup of tasks hanging in the `PENDING` state for a long time. One way to check if the cluster is in this state is to check the cluster's `PendingTasksPerInstance` metric, looking for high averages or a high maximum. The other thing to check is the `WatchbotWorkerPending` metric for all other watchbot stacks. If either of these is high, you are in the throttling scenario.
```

Comment:
Most of this description is Mapbox specific. Can you take a stab at describing the problem and the resolution in a way that doesn't imply the user has the same metrics we may have on some of our clusters?